src/chrome/content/rules/Mod.gov.lv.xml

```diff
@@ -0,0 +1,5 @@
+<ruleset name="Mod.gov.lv">
+  <target host="www.mod.gov.lv" />
+
+  <rule from="^http://www\.mod\.gov\.lv/" to="https://www.mod.gov.lv/" />
```

Comment:
https://www.mod.gov.lv/ doesn't seem to work for me.

Also, you should use the new ruleset format too, especially for a simple rewrite like this